src/internal/storage/fileset/storage.go

```diff
@@ -68,9 +70,9 @@ func NewStorage(store Store, tr track.Tracker, chunks *chunk.Storage, opts ...St
 // TODO Store is just used to poke through the information about file set sizes.
 // I think there might be a cleaner way to handle this through the file set interface, and changing
 // the metadata we expose for a file set as a set of metadata entries.
-func (s *Storage) Store() Store {
-	return s.store
-}
+// func (s *Storage) Store() Store {
```

Comment:
We should remove this if we don't anticipate using it, or at least remove the TODO if we do.